vent/controller/control_module.py

```diff
@@ -558,7 +559,60 @@ def heartbeat(self):
 
 class ControlModuleDevice(ControlModuleBase):
     # Implement ControlModuleBase functions
-    pass
+    def __init__(self):
+        ControlModuleBase.__init__(self)
+        self.HAL = io.Hal(config_file='vent/io/config/devices.ini')
+        self._sensor_to_COPY()
+
+
+    def _sensor_to_COPY(self):
+        # And the sensor measurements
+        self._lock.acquire()
+        self.COPY_sensor_values = SensorValues(pip = self._DATA_PIP,
```

Comment:
maybe want to wait until my PR goes through that uses this

https://github.com/CohenLabPrinceton/Ventilator-Dev/blob/c0c84128f140570e128079e089fa2de4b56b0503/vent/common/message.py#L15

so this method is like this
https://github.com/CohenLabPrinceton/Ventilator-Dev/blob/c0c84128f140570e128079e089fa2de4b56b0503/vent/controller/control_module.py#L716

Reply:
I'm going to pull it in to hardware-dev and flesh out the stubs a bit before merging into master, so I think that should be fine.

Reply:
good point! Yep, we should wait then